load_balancers.go

```diff
@@ -95,7 +96,7 @@ type ForwardingRule struct {
 	TargetProtocol string `json:"target_protocol,omitempty"`
 	TargetPort     int    `json:"target_port,omitempty"`
 	CertificateID  string `json:"certificate_id,omitempty"`
-	TlsPassthrough bool   `json:"tls_passthrough,omitempty"`
+	TLSPassthrough bool   `json:"tls_passthrough,omitempty"`
```

Comment:
Same here – sorry about all of these, didn't realize how many of these there were in our codebase.